src/plugins/embeddable/public/lib/containers/container.ts

```diff
@@ -301,7 +301,7 @@ export abstract class Container<
 
   public async getExplicitInputIsEqual(lastInput: TContainerInput) {
     const { panels: lastPanels, ...restOfLastInput } = lastInput;
-    const { panels: currentPanels, ...restOfCurrentInput } = this.getInput();
+    const { panels: currentPanels, ...restOfCurrentInput } = this.getExplicitInput();
```

Comment:
**Note:** Since the navigation embeddable is no longer a container, this bug no longer impacts it; however, I figured I might as well keep this fix.

This was a bug that was never caught because we've never had a container embeddable as a panel in Dashboard before - basically, because we were calling `getInput` here instead of `getExplicitInput`, the dashboard **always** had unsaved changes back when the navigation embeddable was a container :)